ISISScientificComputing/autoreduce
# extend setup.py

**Description of changes**

* Remove multiple .sh/.bat scripts in favour of python scripts where possible
* Add several commands to the setup.py to allow for project build steps (e.g. build database, migrate test files etc.)
* Updated travis to run new script
* Pylint changes to multiple files
* update wiki pages


**How to test**
* Follow the new [installation guide](https://github.com/ISISScientificComputing/autoreduce/wiki/Installation-for-local-development)
* Try to use all the commands with the new `setup.py` 
  * `python setup.py help` will display all of these
* Try these out on both windows and linux - ideally in a clean environment to check the installation work correctly
* Check the logs `build.log` look reasonable
* Check the build runs successfully on travis


Fixes #70
